src/dataHelpers.jl

```diff
@@ -166,7 +170,7 @@ function importHumanized(dataType)
         @error "Data type not found"
```

Comment:
@cyrillustan remaining issue seems to be that `schwab_ITP_humanized.csv` isn't joining successfully with the Fc affinities.